verify-merge.py

```diff
@@ -15,8 +15,13 @@ def verify():
         import_gpg_keys()
     if args.refresh_keys:
         refresh_gpg_keys()
-    assert_files = get_assert_file_list()
-    verify_gpg_sigs(assert_files)
+    # Shell glob pattern for specific version or all builds:
+    ver_pattern = args.version if args.version else 'v0*'
+    sig_file_paths = set(glob.glob(ver_pattern + '-*/*/*.assert.sig'))
+    assert_files = get_assert_file_list(ver_pattern)
```

Comment:
Why the distinguish between `assert_files` and `sig_file_paths`? Can't we get both of them from a common function given the version as an argument?

Reply:
`assert_files` is a dict of path attributes for the assert files.
`sig_file_paths` is just a list of paths to signature files.

We need them seperate so we can detect missing sig files for asserts or assert files for sigs.
Does that help?

Reply:
I see, all good then.